java/registry/src/main/java/io/opensaber/registry/helper/RegistryHelper.java

```diff
@@ -61,23 +61,20 @@
      * @return
      * @throws Exception
      */
-    public String addEntity(JsonNode inputJson, String userId) throws Exception {
-        RecordIdentifier recordId = null;
-        String entityType = inputJson.fields().next().getKey();
-        String jsonString = objectMapper.writeValueAsString(inputJson);
+    public JsonNode addEntity(JsonNode rootNode, String userId) throws Exception {
```

Comment:
Didn't your assertion that "no need to return because rootNode will have the fields" work?

Reply:
Yes the rootNode will be updated with the new fields.